app/views/registrationComplete.scala.html

```diff
@@ -0,0 +1,19 @@
+@(name: String)(implicit request: RequestHeader, messages: Messages)
```

Comment:
This all look fine. I'm just wondering about the links to the FAQs and Help pages. I think we should have different pages for the judgments. It might just be worth following up with Kirsten to double check this. If so I think we will need some tasks to update the controllers for these pages, if we don't have so already.

Reply:
The new designs for the FAQ and Help pages for judgment still need to be reviewed & implemented. I can edit the links to those once they are done.